src/v3/test/integration/authenticator-enroll-security-question.test.tsx

```diff
@@ -119,16 +119,16 @@ describe('authenticator-enroll-security-question', () => {
         expect.anything(),
       );
       // assert global alert
-      const globalError = await findByRole('alert');
-      expect(globalError.innerHTML).toContain('We found some errors. Please review the form and make corrections.');
+      const globalError = await findAllByRole('alert');
+      expect(globalError[0].innerHTML).toContain('We found some errors. Please review the form and make corrections.');
```

Comment:
nit: is there any way to improve the `findByRole` with options to single out only one element? if not, you can also use array destructuring to pull the first value so that `globalError` still represents a single value and not an array.